website/docs/cloud-docs/api-docs/run-tasks/run-task-stages-and-results.mdx

```diff
@@ -0,0 +1,259 @@
+
+# Run Task Stages API
+
+Each run passes through several [stages](/cloud-docs/run/states) of action, and Terraform Cloud shows the progress through those stages as run states.
+Run task stages are actioned as the run processes the different stages, namely;
+
+- [Post-Plan Stage](/cloud-docs/run/states#3-the-post-plan-stage) which executes [run tasks](/cloud-docs/workspaces/settings/run-tasks) after the plan is complete.
+
+-> **Note:** Run Tasks is a paid feature, available as part of the **Team & Governance** upgrade package. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
+
+## Attributes
+
+### Task Stage States
+
+The task stage state is found in `data.attributes.status`, and you can reference the following list of possible states.
+
+| State         | Description                                                  |
+|---------------|--------------------------------------------------------------|
+| `pending`     | The initial status of a task stage once it has been created. |
+| `running`     | The task stage is running.                                   |
+| `passed`      | All of the results in the task stage passed.                 |
+| `failed`      | One more results in the task stage failed.                   |
+| `errored`     | The task stage has errored. This is a final state.           |
+| `canceled`    | The task stage has been canceled.                            |
+| `unreachable` | The task stage could not be executed.                        |
+
+### Task Result States
+
+The task result state is found in `data.attributes.status`, and you can reference the following list of possible states.
+
+| State         | Description                                                   |
+|---------------|---------------------------------------------------------------|
+| `pending`     | The initial status of a task result once it has been created. |
+| `running`     | The task result is running.                                   |
+| `passed`      | All of the results in the task result passed.                 |
+| `failed`      | One more results in the task result failed.                   |
+| `errored`     | The task result has errored. This is a final state.           |
+| `canceled`    | The task result has been canceled.                            |
+| `unreachable` | The task result could not be executed.                        |
+
+## List Task Stages in a Run
+
+`GET /runs/:run_id/task-stages`
+
+| Parameter | Description                         |
+|-----------|-------------------------------------|
+| `run_id`  | The run ID to list task stages for. |
+
+| Status  | Response                                                | Reason                          |
+|---------|---------------------------------------------------------|---------------------------------|
+| [200][] | Array of [JSON API document][]s (`type: "task-stages"`) | Successfully listed task-stages |
```

Comment:
the trailing `s` outside of the JSON API document array seems odd.

Reply:
![Screen Shot 2022-06-14 at 1 18 18 pm](https://user-images.githubusercontent.com/186643/173486274-b0672781-8af1-4863-8306-1bab84071275.png)

Reply:
Huh so..

1. How on earth did you spot that? 🦅👀!
2. This is a copy-pasta from run and ssh-keys docs.  They have the same trailing 's'

Looks like it's needed to have a consistent reference to 'JSON API Document'.  But I'll change that to use 'documents' appropriately.

Reply:
I think `Array of [JSON API document][]` is fine. Just remove the s.

Reply:
Changed to have a singular and plural link. Reads better with an s